main.go

```diff
@@ -82,6 +82,672 @@ var (
 	masterMetrics = []snapshotMetric{
 		snapshotMetric{
 			desc: prometheus.NewDesc(
+				"mesos_master_cpus_percent",
+				"Percent of CPUs used as a fraction of 1.",
```

Comment:
I've tried to keep original names from Mesos stats. Do we care about that?

I can fix description in any case.